### ISSUE TITLE
Toonify needs to avoid adding hair to images of bald people

### ISSUE THREAD

**Issue by jawhnycooke** [justinpinkney/toonify-api-docs#1]
You may want to add a check box for people who don't have hair. The current results are not good when the supplied image has a bald person
![Screen Shot 2020-10-22 at 10 29 59 AM](https://user-images.githubusercontent.com/11808002/96908492-920a7f00-1451-11eb-8fed-e77cf4742082.png)

**Comment by justinpinkney** [comment]
Yes this is currently a limitation of the model. It also like to turn hats into hair too. I have some ideas how to improve it, but it's not straightforward.
